2018AsianGamesApp/app/src/main/java/com/example/mrtbkwjn/a2018asiangamesapp/LoginActivity.java

```diff
@@ -0,0 +1,71 @@
+package com.example.mrtbkwjn.a2018asiangamesapp;
+
+import android.app.ActionBar;
+import android.content.Intent;
+import android.os.Build;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.View;
+import android.view.WindowManager;
+import android.widget.Button;
+import android.widget.ImageButton;
+
+public class LoginActivity extends AppCompatActivity {
+
+    private ImageButton googleButton = (ImageButton) findViewById(R.id.glogButton);
```

Comment:
This is subjective, but I prefer the id name is not shortend. So `googleLoginButton` or `facebookLoginButton` or `FBLoginButton` are my preference. 

I'd argue that it makes reading the code more easy and does not take mental effort to understand what this ID is for. But this is purely subjective matters